### ISSUE TITLE
dyld: Symbol not found: _MTLHeapDescriptorClass

### ISSUE THREAD

**Issue by chatsopond** [lupidan/apple-signin-unity#10]
I try to build your sample project and I can build without any error until the program starts, the Xcode shows an error

Thank for your great plugin but Am I missing something?

> dyld: Symbol not found: _MTLHeapDescriptorClass
>   Referenced from: /private/var/containers/Bundle/Application/B0273215-7B22-4F1A-9FE6-A2CA022CB61D/testappleauth.app/testappleauth
>   Expected in: flat namespace
>  in /private/var/containers/Bundle/Application/B0273215-7B22-4F1A-9FE6-A2CA022CB61D/testappleauth.app/testappleauth
> (lldb) 

<img width="631" alt="ภาพหน้าจอ 2562-12-10 เวลา 23 59 14" src="https://user-images.githubusercontent.com/42887325/70550925-2147d480-1ba9-11ea-9166-4ebae6b5b0aa.png">

**Comment by chatsopond** [comment]
By the way I create a new project and then import your project as a package. It can run normally.

**Comment by lupidan** [comment]
Thanks for taking the time to open an issue, but I'm confused as to why this issue was closed.

Is this still happening with the sample project? Or was it just a bad Unity build?

**Comment by chatsopond** [comment]
This issue will be found when I open the Sample Project. It will not happen if I open the Sample Project, then export the Asset folder as the package and import it to the empty project. 

It may cause some previous build settings about simulator SDK (I just found a minute ago) because I ran it right to my device. If this setting be device SDK, it should be normal.

<img width="335" alt="ภาพหน้าจอ 2562-12-11 เวลา 03 50 52" src="https://user-images.githubusercontent.com/42887325/70568033-a347f580-1bc9-11ea-80ac-b5d66ddabfbb.png">

I closed this issue because default build settings can normally run. It may waste your time to fix this because it's not the issue about your code.

Thank for your response and helpful.
